modules/nw-nad-benefits.adoc

```diff
@@ -0,0 +1,21 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-multiple-networks.adoc
+:_mod-docs-content-type: REFERENCE
+[id="nw-nad-benefits_{context}"]
+= Benefits of an additional network created by a NAD
+
+You can use an additional network in situations where network isolation is needed, including data plane and control plane separation. Isolating network traffic is useful for the following performance and security reasons:
+
+. Performance
++
+**Traffic management**: You can send traffic on two different planes to manage how much traffic is along each plane.
+
+. Security
++
+**Network isolation**: You can send sensitive traffic onto a network plane that is managed specifically for security considerations, and you can separate private data that must not be shared between tenants or customers.
+
+All of the pods in the cluster still use the cluster-wide default network to maintain connectivity across the cluster. Every pod has an `eth0` interface that is attached to the cluster-wide pod network. You can view the interfaces for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you add additional network interfaces that use Multus CNI, they are named `net1`,
+`net2`, ..., `netN`.
+
+To attach additional network interfaces to a pod, you must create configurations that define how the interfaces are attached. You specify each interface by using a `NetworkAttachmentDefinition` custom resource (CR). A CNI configuration inside each of these CRs defines how that interface is created.
```

Comment:
Added additional resources to point to the sections specifically for a better user experience. I think it is outside the scope of this PR to update the NAD docs. However, I take your point. I don't think we want a best practices at all in the NADs docs since we want users, ideally, to be using UDN. Perhaps, limitations could be helpful in the latter's efforts. Could be contentX work and we could do it after the support matrix merges. #81484